README.md

```diff
@@ -2,14 +2,23 @@
 
 ## Description
 
-This is the official Helm Chart for [OpenStack Exporter](https://github.com/openstack-exporter/openstack-exporter), a tool to export Prometheus metrics from a running OpenStack Cloud.
+This is a forked version of the [official Helm Chart](https://github.com/openstack-exporter/helm-charts) for [OpenStack Exporter](https://github.com/openstack-exporter/openstack-exporter), a tool to export Prometheus metrics from a running OpenStack Cloud.
+The fork has been created due to the inactivity of the official Helm Chart, see [#21](https://github.com/openstack-exporter/helm-charts/issues/21).
 
 ## Configuration
 
 The chart configuration is done in the `values.yaml` file.
 
 ## Usage
 
+### Installation from helm registry
+The most recent versions of Helm chart packages are published in the [SCS registry](https://registry.scs.community/harbor/projects/31/repositories/prometheus-openstack-exporter/artifacts-tab).
```

Comment:
As `helm search` command doesn't work yet for oci registries, I suggest adding `?publicAndNotLogged=yes` so the public can search for available versions without the need for sign-in:
```suggestion
The most recent versions of Helm chart packages are published in the [SCS registry](https://registry.scs.community/harbor/projects/31/repositories/prometheus-openstack-exporter/artifacts-tab?publicAndNotLogged=yes).
```